### PR TITLE
(MAINT) Add Travis CI support

### PR DESCRIPTION
**Pull request by camlow325**
This commit adds Travis CI support and a Travis CI Build Status link to the README.md.

### CODE REVIEW

**Comment by fpringvaldsen** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: clojure
+lein: lein2
+jdk:
+  - oraclejdk7
+  - openjdk7
```
Comment (KevinCorcoran) [inline .travis.yml]: We've been including oraclejdk8 elsewhere ... seems like a good idea to do that here too?

File: ext/travisci/test.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+lein2 test :all
```
Comment (KevinCorcoran) [inline ext/travisci/test.sh]: `:all` is unnecessary (although certainly doesn't hurt anything)

**Comment by KevinCorcoran** [comment]
:+1: aside from adding Java 8 (if we think that's worthwhile)

**Comment by camlow325** [comment]
Thanks, @KevinCorcoran.  I agreed with both of your comments so I squashed those into a new commit, b494315.

**Pull request merged**